processing/process_data.sh

```diff
@@ -78,7 +78,8 @@ segment_if_does_not_exist(){
   local contrast="$2"
   # Update global variable with segmentation file name
   FILESEG="${file}_seg"
-  FILESEGMANUAL="${PATH_SEGMANUAL}/${FILESEG}-manual.nii.gz"
+  FILESEGMANUAL="${PATH_DATA}/derivatives/${SUBJECT}/anat/${FILESEG}-manual.nii.gz"
```

Comment:
Shouldn't path change based on anat or dwi?, i.e. 
${PATH_DATA}/derivatives/${SUBJECT}/`anat`/${FILELABEL}-manual.nii.gz
or
${PATH_DATA}/derivatives/${SUBJECT}/`dwi`/${FILELABEL}-manual.nii.gz

Reply:
ah! great catch @valosekj. thanks! will fix right away

Reply:
Fixed in 1e07ffd

Reply:
actually, this one is better: f8021a99cb3bc1b9d2294d2294bcdac7d09ee687